protoc-gen-swagger/options/openapiv2.proto

```diff
@@ -181,8 +226,8 @@ message Response {
   // `Schema` optionally defines the structure of the response.
   // If `Schema` is not provided, it means there is no content to the response.
   Schema schema = 2;
-  // field 3 is reserved for 'headers'.
-  reserved 3;
+  // `Headers` A list of headers that are sent with the response.
```

Comment:
We should define what the format of the string key should be. Generally in Go we use the format produced by https://golang.org/pkg/net/textproto/#CanonicalMIMEHeaderKey for header keys.

Reply:
Do we want to fail if it is not provided in that format? or just convert it using that function ☝️